dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.idl

```diff
@@ -5,9 +5,56 @@ import "..\AppNotifications\AppNotifications.idl";
 
 namespace Microsoft.Windows.AppNotifications.Builder
 {
-    [contractversion(1)]
+    [contractversion(2)]
     apicontract AppNotificationBuilderContract {}
 
+    [contract(AppNotificationBuilderContract, 2)]
+    runtimeclass AppNotificationHeader
+    {
+        AppNotificationHeader();
+
+        String Id;
+        String Title;
+        Windows.Foundation.Collections.IMap<String, String> Arguments;
+
+        AppNotificationHeader(String id, String title);
+
+        AppNotificationHeader AddArgument(String key, String value);
+    };
+
+    [contract(AppNotificationBuilderContract, 2)]
+    enum AppNotificationTextStyle
+    {
+        Default,
+        Caption,
+        CaptionSubtle,
+        Body,
+        BodySubtle,
+        Base,
+        BaseSubtle,
+        Subtitle,
+        SubtitleSubtle,
+        Title,
+        TitleSubtle,
```

Comment:
What is the difference between (let's say) normal, subtle and numeral?

Reply:
Numeral: Same as normal but with top/bottom padding removed.
Subtle: Same as normal but with subtle opacity.

Reply:
Doc-comments would make this intuitively obvious to the most casual of readers... ;-)